### DISCUSSION TITLE
Another way to progress through passages

### DISCUSSION THREAD

**Discussion by ShieldyArmor** [FakeFishGames/Barotrauma#10161] [Ideas and suggestions]
Thought it would be cool to be able to force yourself through by killing the outpost officer and doing some sequence using his ID card to open the passage illegally or straight up having to destroy/force open a huge mechanical door on the right of the outpost for free to sneak through though loosing a lot of rep for the outpost and potentially faction, or being flat out able to just fight through the Outpost till it is completely wiped out to pass through.

This could make "pirate" runs more exciting as you can fully act on yourself in this case, also if your rep is low enough to the point you get shot and NPCs do not interact with you, this would avoid a soft-lock, which is possible either if you cannot go back (due to jovian radiation or can't come back in any reasonable amount of time due to being unable to take missions in consistently.

**Comment by Petethegoat** [comment]
Having a choice between getting enough reputation, bribery, forcing your way through, (or maybe finding a different hidden passage?) would be very cool.
Right now it's quite hard to even get to the passage without enough rep unless you're trying to lose it, but especially when reputation is more balanced in the future this would be a great addition.

**Comment by MarkCringan** [comment]
I feel like to give the other factions better uses they could provide you alternative ways of progressing. Maybe Jovian separatists control a longer and more dangerous passageway (+20% normal difficulty?) in order to go between biomes?

**Comment by Ek-Videogames** [comment]
The idea was tossed around a bit when discussing faction reputation (Hey devs, when's faction rep going to actually matter btw?) - such as the possibility of using connections with the other resistance factions to let you lean about some sort of secret smuggling route.

**Reply by MapleWheels** [reply]
> The idea was tossed around a bit when discussing faction reputation (Hey devs, when's faction rep going to actually matter btw?) - such as the possibility of using connections with the other resistance factions to let you lean about some sort of secret smuggling route.

For now, literally just a mod that locks a route behind Jovian separatist rep. Then the full implementation is just having some outposts be Jovian-controlled. You can even end up taking missions to raid another outpost, husk it, etc. Would be cool...
